charters/DataPortal/charter.md

```diff
@@ -0,0 +1,78 @@
+* Registration form for tertiary portals and list of portals available with click through to hosting web site (e.g. Dockstore)
+* Integration with help desk, discussion forum, and user feedback applications
+### Data Browser
+* Create (or use Data Storage Service [DSS] provided) indexes to access files in the Data Store by using public DSS APIs to read the metadata
+* Browser support for multiple metadata schemas that are supported by the schema access library maintained by the Ingestion Service
+* Search views of Specimens, Projects and Files
+* Support for specifying multiple search criteria (“facets”) through the Browser UI
+* Download of selected files from the Data Store via the UI and/or HCA DCP Command Line Interface (CLI)
+* Browser interface to the Date Store Collections Service to save, view, list, share and delete the query results from a search operation
+* Search results / Collections interface to Analysis Applications. Provide UI for handoff of search results to FireCloud.
+* Facilities to save, view, share, update, and delete search queries and personal Collections on a per-user basis
+* Search results / Collections interface to the Matrix Service
+* API Documentation for portable, cloud-neutral APIs
+
+## Out-of-scope
+* Production operation of applications and services linked to from the Data Portal (e.g. the Data Storage Service, Ingestion Infrastructure, Upload Service, Data Processing Pipelines, Analysis Applications, etc.)
+* Any separate Ingestion Service application which may replace or augment the "Contribute" tab of the Data Portal
+* Methods Registry providing management, searching, filtering, sorting, visualization, and methods packages in the "Analyze" tab of the Data Portal
+* Primary interface for collecting user feedback and feature/bug requests
+
+## Milestones and Deliverables
+* Oct 2018:  Deploy a minimum viable product (MVP) feature set as part of HCA DCP beta test phase
+* Nov 2018:  Present beta phase portal to the DCP community
+* EOY 2018:  Implement as needed improved scaling/hardening to address any gaps identified during the HCA DCP beta test phase
+* 1H  2019:  Save search queries and query results (using DSS Collections APIs). Search results / Collections interface to the Matrix Service and to Analysis Applications. Discussion forums support. User feedback application support.
+
+## Roles
+
+### Project Lead
+[Brian O’Connor](mailto:brocono@ucsc.edu) 
+
+### Product Owner
+[Trevor Heathorn](mailto:theathor@ucsc.edu) 
+
+### Technical Lead
+Portal: [Dave Rogers](mailto:dave@clevercanary.com)
+Browser: [Hannes Schmidt](mailto:hannes@ucsc.edu)
+
+## Communication
+
+### Slack Channels
+* HumanCellAtlas/data-portal: general data portal discussions
+* HumanCellAtlas/data-portal-dev: data portal development discussions
+* HumanCellAtlas/content-team: data portal content development discussions
+* HumanCellAtlas/data-browser-dev: data browser development discussions
```

Comment:
Is data-browser-dev a closed channel? Will there be an open, non-dev, channel like for the Data Portal but for the Data Browser, where people can ask general questions about it?

Reply:
Working on making data-browser-dev a public channel. data-browser is already a public channel.

Reply:
I don't see #data-browser in the HCA Slack group. Is it perhaps called something different? I think it's ok if #data-browser-dev is meant to be a closed channel for developers only. I just can't see any channels with "browser" in the name in the HCA Slack group, and it would be good to have at least 1 that is public or that at least other DCP members can be a part of.